WatchGuard patches to sources for pam-1.3.0 as of
Thu May 26 11:38:00 PDT 2022
The patches shown here have been applied to source .tar.gz 
files supplied with the WatchGuard Open Source Archive.

==========================================================================
--- pam-1.3.0/build-aux/ltmain.sh.orig	2022-05-26 11:37:53.487944999 -0700
+++ pam-1.3.0/build-aux/ltmain.sh	2022-05-26 11:37:53.907927786 -0700
@@ -6918,7 +6918,8 @@
 	      fi
 	    else
 	      # We cannot seem to hardcode it, guess we'll fake it.
-	      add_dir="-L$libdir"
+	      # WG: don't fake it this introduces host pollution
+	      # add_dir="-L$libdir"
 	      # Try looking first in the location we're being installed to.
 	      if test -n "$inst_prefix_dir"; then
 		case $libdir in
